app/v3/models/request/createForeignPropertyPeriodSummary/foreignFhlEea/ForeignFhlEeaIncome.scala

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package v3.models.request.createForeignPropertyPeriodSummary.foreignFhlEea
+
+import play.api.libs.json.{Json, Reads, Writes}
+
+case class ForeignFhlEeaIncome(rentAmount: Option[BigDecimal])
+
+object ForeignFhlEeaIncome {
+  implicit val reads: Reads[ForeignFhlEeaIncome]   = Json.reads[ForeignFhlEeaIncome]
+  implicit val writes: Writes[ForeignFhlEeaIncome] = Json.writes[ForeignFhlEeaIncome]
```

Comment:
Could simplify this by replaceing with format
```suggestion
  implicit val format: Format[ForeignFhlEeaIncome] = Json.format
```